### PR TITLE
yubikey fix + vpn script

### PR DESCRIPTION
**Pull request by gvacaliuc**
See README.md

You should be able to run `bin/vpn` to start the vpn.  If you have a yubikey
you can run `GP_MFA_ORDER=webauthn bin/vpn`.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,32 @@
+# Usage at GR
+
+You should be able to start the vpn by running `bin/vpn`.  You can add
+"$GR_HOME/pan-globalprotect-okta/bin" to your path to start the vpn from
+anywhere.
+
+There are two everironment variables you may want to configure:
```
Comment (cobbr2) [inline README.md]:
```suggestion
There are two environment variables you may want to configure:
```

File: bin/vpn [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+GP_USERNAME=${GR_USERNAME:?"Must add GR_USERNAME to .localrc"}
+GP_GATEWAY=${GP_GATEWAY:-"US West"}
+
+GP_PASSWORD_STORE_PATH="$HOME/.globalprotect-password.gpg"
+GP_PASSWORD=
+[[ -f "${GP_PASSWORD_STORE_PATH}" ]] && GP_PASSWORD=$(gpg -dq "$GP_PASSWORD_STORE_PATH")
+if [[ -z "${GP_PASSWORD}" ]]; then
+    read -s -p "GP Password: " GP_PASSWORD
+    echo "INFO: storing GP Password in encrypted file: ${GP_PASSWORD_STORE_PATH}"
+    echo "$GP_PASSWORD" | gpg -e -r "$GR_GPGKEY_ADDRESS" > "$GP_PASSWORD_STORE_PATH"
```
Comment (cobbr2) [inline bin/vpn:13]: What should `$GR_GPGKEY_ADDRESS` be? I just tried running this, and got:
```
rcobb@rcobb-t480: (gabe/upstream-plus-yubikey) ~/pan-globalprotect-okta$ bin/vpn
GP Password: INFO: storing GP Password in encrypted file: /home/rcobb/.globalprotect-password.gpg
bin/vpn: line 14: GR_GPGKEY_ADDRESS: unbound variable
```

Comment (cobbr2) [reply bin/vpn:13]: Perhaps I've missed some new onboarding scripts?

**Review by cobbr2 — COMMENTED** [review]
This review is out-dated, but I don't see a way to retract it and do another; will produce another one shortly.

File: bin/vpn (hunk 2 above)
Comment (cobbr2) [reply bin/vpn:13]: Turned out I did, so this isn't a problem.

**Review by cobbr2 — APPROVED** [review]
Well, I'm done suggesting changes via commit.  How about you do a final review & merge, @gvacaliuc ?

**Comment by gvacaliuc** [comment]
LGTM, thanks for adding the username bits

**Pull request merged**